docs/services/slam/cartographer/_index.md

```diff
@@ -13,93 +13,107 @@ aliases:
 
 [The Cartographer Project](https://github.com/cartographer-project) contains a C++ library that performs dense SLAM.
 
-Viam provides the `cartographer` [modular resource](/modular-resources/) which adds support for using Cartographer with the Viam [SLAM service](/services/slam/).
-
-Since creating maps with Cartographer is CPU-intensive, for **creating** or **updating** a map, the `cartographer` modular resource **runs in the cloud**.
+To use Cartographer with the Viam [SLAM service](/services/slam/), you can use the [`cartographer`](https://app.viam.com/module/viam/cartographer) [modular resource](/modular-resources/).
+See [Modular resources](/modular-resources/#the-viam-registry) for instructions on using a module from the Viam registry on your robot.
 
-For doing **pure localization** on an existing map, the `cartographer` modular resource **runs on your robot**.
+The source code for this module is available on the [`viam-cartographer` GitHub repository](https://github.com/viamrobotics/viam-cartographer).
 
 {{% alert title="Info" color="info" %}}
 
-See Viam's [Pricing](https://www.viam.com/product/pricing) page to understand the costs associated with running Cartographer in the cloud.
+Cartographer supports taking **2D LiDAR** or **3D LiDAR** data and optionally **inertial measurement unit (IMU)** and/or **odometry** data as input.
+
+However, currently, the `cartographer` modular resource only supports taking **2D LiDAR** and optionally **IMU** data as input.
+Support for taking **3D LiDAR** and **odometry** data as input may be added in the future.
 
 {{% /alert %}}
 
-The `cartographer` {{< glossary_tooltip term_id="module" text="module" >}} is available [from the Viam registry](https://app.viam.com/module/viam/cartographer).
-See [Modular resources](/modular-resources/#the-viam-registry) for instructions on using a module from the Viam registry on your robot.
+## Requirements
```

Comment:
No, I don't think that makes sense. These requirements are still required to get the data to do mapping in the first place.

The UI as is presents what you refer to as online and offline operations as entirely different flows. While that technically may use the same service

Reply:
Ok, that makes sense that you need to have captured the data at some point. Sounds good.

Reply:
Whoops, I didn't see your second comment. I think either would make sense - having a separate requirements section for live robot and offline, or having this requirement at the top because it technically applies to both.

Reply:
(Unresolving since I had resolved - feel free to resolve)